build/webpack-watch.js

```diff
@@ -10,6 +11,16 @@ let server;
 
 module.exports = config => {
 
+  // normalize config output paths
+  // (code sniped from bin/webpack-dev-server.js:95)
+  // without this, assets are not loaded correctly
+  [].concat(config).forEach(function(wpOpt) {
+    wpOpt.output.path = "/";
+  });
+
```

Comment:
looping/accessing objects in this fashion is confusing.  It requires you to understand that objects are passed by reference in js and you're causing a side effect.  Prefer assigning a map when possible, or assigning via index.

```
config = config.map(wpOpt => {
  wpOpt.output.path = '/';
  return wpOpt;
});
```

```
config.forEach((_, i) => {
  config[i].output.path = "/";
});
```